thrones/db/seeds.rb

```diff
@@ -0,0 +1,32 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+#   Character.create(name: 'Luke', movie: movies.first)
+require_relative './house_data.rb'
+require_relative './character_data.rb'
+
```

Comment:
this looks right to me, but i wasn't able to get your seed data to load. Did this work on your machine?